tests/pipelines/brain_observatory/test_mesoscope_split_z.py

```diff
@@ -0,0 +1,336 @@
+import pytest
+import tifffile
+import numpy as np
+import h5py
+import os
+import copy
+from ophys_etl.transforms.mesoscope_2p import MesoscopeTiff
+from ophys_etl.pipelines.brain_observatory.scripts.run_mesoscope_splitting import split_z  # noqa: E501
+
+
+"""
```

Comment:
this is not an ideal place to document at length. A user looking for information by looking in the code won't naturally look in the test.
Can this be moved somewhere in the code itself.
Would creating explicit data structures eliminate the need for some/all of this text?

Reply:
@danielsf

Reply:
Feel free to move it wherever you feel is appropriate. I just wanted to write it down before I forgot.

If we are going to use this ticket to fill in all the docstrings, this should probably go in the `split_z` function. Though, I would argue that it should be a comment in the code, rather than a part of the docstring. As a user-qua-user, I would probably not appreciate a docstring that was that long. It mostly needs to be there for future developers.

Reply:
Honestly I think this should go in a confluence page. It's just too big for code, I think. Maybe even add a comment into the code with a link to a confluence page?

Reply:
how about a README file?

Reply:
Ooh that's a good idea!